spec/schemas/ciag-induction/ciag-induction-created.yml

```diff
@@ -0,0 +1,55 @@
+name: ciag-induction.created
+title: CIAG Induction Created
+summary: A CIAG induction has been created for a Prisoner.
+contentType: application/json
+payload:
+  type: object
+  properties:
+    eventType:
+      type: string
+      example: "ciag-induction.created"
```

Comment:
Given that this is the schema for one specific event type, the example in this case feels like less of an example, and more of the 1 and only 1 value that is supported here?
Could/should this be something like:
```
properties:
  eventType:
    type: string
    enum:
      - "ciag-induction.created"
```

Would that work?

Reply:
Good idea. Done.